oaknational/Oak-Web-Application
# PUPIL-632 feat(unitlisting): create unit listing page

## Description

Music year: 1972

create boiler plate for unit listing page

- [x]  Served at pupils/beta/programmes/PROGRAMME_SLUG/units
- [x]  Displays subject title, year and programme factors
- [x]  List units in correct order as links to lesson listing page
- [x]  Lists number of lessons against each lesson listing
- [x]  Back button goes to programmes or subject page according to programme factors
- [x]  Back button adds query string of those programme factors which aren’t being modified. (eg. examboard=aqa)
- [x]  Back button displays “select tier”, “select examboard” or “select subject” according to programme factors present. If tiers come ahead of examboard.

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2391--oak-web-application.netlify.thenational.academy/pupils/beta/programmes/biology-secondary-year-10-foundation-edexcel/units
2. Go to (https://deploy-preview-2391--oak-web-application.netlify.thenational.academy/pupils/beta/programmes/citizenship-secondary-year-7-l/units)
3. Click on unit links and back links
4. You should be taken to the lesson listing pages and previous pupils journey page respectively

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
